pkgs/servers/bugzilla/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, pkgs, fetchurl, perlPackages }:
+
+stdenv.mkDerivation rec {
+
+    name = "bugzilla-5.1.1";
+
+    src = fetchurl {
+        url = "https://ftp.mozilla.org/pub/webtools/${name}.tar.gz";
```

Comment:
maybe it would be better to fetch from github directly https://github.com/bugzilla/bugzilla